lib/matplotlib/scale.py

```diff
@@ -667,14 +667,31 @@ class LogitScale(ScaleBase):
     """
     name = 'logit'
 
-    def __init__(self, axis, nonpos='mask'):
-        """
+    def __init__(
+        self,
+        axis,
+        nonpos='mask',
+        *,
+        one_half=r"\frac{1}{2}",
+        use_overline=False,
+    ):
+        r"""
         *nonpos*: {'mask', 'clip'}
           values beyond ]0, 1[ can be masked as invalid, or clipped to a number
           very close to 0 or 1
+
+        use_overline: bool (default: False)
+            indicate the usage of survival notation (\overline{x}) in place of
+            standard notation (1-x) for probability close to one.
+
+        one_half : str (default: r"\frac{1}{2}")
+            the string used for ticks formatter to represent 1/2. If None,
+            "0.5" is used.
```

Comment:
Again I wouldn't bother with supporting None here; but if you do want to support None, then it should match the default (that's the typical Matplotlib behavior), so "r\frac{1}{2}".

Reply:
None is not supported now. Fixed.